util/binfmt_misc/fixtures/exit.ppc64le.s

```diff
@@ -0,0 +1,7 @@
+	.global _start
+	.text
+	.abiversion 2
```

Comment:
😠

Reply:
?

Reply:
@AkihiroSuda it took us a while to realize that ppc64 had two ABIs and that we needed this directive.